src/Bundle/Resources/skeleton/Factory.tpl.php

```diff
@@ -50,7 +50,7 @@ protected function initialize(): self
         ;
     }
 
-    protected static function getClass(): string
+    public static function getClass(): string
```

Comment:
Not happy about that change. Maybe you have a better idea how we can access the entities class name?